ansible-tests/validations/network_environment.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: undercloud
+  become: false
```

Comment:
I'd like our ansible default to be false, and explicitly use 'become' when it is required. Can we do that?